### ISSUE TITLE
editorial intrusions into 1 h.1

### ISSUE THREAD

**Issue by dietmarpravida** [faustedition/xml#148]
Bei 1 H.1 3095 wird nach "Drum" unisnnigerweise ein editorisch ergänztes Komma angezeigt:

http://dev.faustedition.net/print/Cotta_Ms_Goethe_AlH_C-1-12_Faust_I.16#l3095

Solche Eingriffe haben in H.1 nichts verloren.

**Comment by gerritbruening** [comment]
Dann auch gleich aus der Kodierung rausnehmen?  Allerdings,  es ist auch nicht nichts da.

**Comment by dietmarpravida** [comment]
Letztlich ist H.1 eine Handschrift, dass in dem zugrundeliegenden Druck ein Buchstabe ausgefallen ist, ist eine Info für Drucke und sollte in der Datei für B stehen, oder? Andernfalls sollte nur der Zeichenausfall (bzw. ein zusätzliches Leerzeichen) kodiert weden. Dumm ist allerdings, dass es dann für oberflächliche Nursohingucker in der Synopse so aussehen könnte, als sei ein Komma gestrichen worden.

**Comment by gerritbruening** [comment]
Ja, hier wird ganz streng ein Exemplar ediert und nicht der "Druck", der durch die Gesamtheit der Exemplare repräsentiert wird.
Zwei Leerzeichen helfen nichts, denn sie gelten im XML genauso viel wie eines. Aber vielleicht ist es auch OK, das Komma einfach wegzulassen.

**Comment by dietmarpravida** [comment]
````
<sp>
         <speaker n="before_3095" rend="centered letter-spaced">Marthe.</speaker>
              <l n="3095">Drum<!-- <supplied reason="printing-error">,</supplied> -->
                                    werther Herr, berathet euch in Zeiten.</l>
</sp>
````

so okay?

**Comment by gerritbruening** [comment]
Gibt es hier weitere Fälle?
@dietmarpravida, sage in der Datei doch mal
`//supplied`
![grafik](https://user-images.githubusercontent.com/10358005/46408621-fe06d700-c712-11e8-907f-20d6dc4a68e6.png)
und Enter
